File: Spring/haru/src/main/java/com/ssafy/haru/model/dto/Diary.java
package com.ssafy.haru.model.dto;

import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class Diary {
    private int diaryNo;
    private String userId;
    private String diaryEmotion;
    private String diaryContent;
    private String diaryDate;

    @Override
    public String toString() {
        return "Diary{" +
                "diaryNo=" + diaryNo +
                ", userId='" + userId + '\'' +
                ", diaryEmotion='" + diaryEmotion + '\'' +
                ", diaryContent='" + diaryContent + '\'' +
                ", diaryDate='" + diaryDate + '\'' +
                '}';
    }
}

File: android/app/src/main/java/com/ssafy/diary/dto/Diary.kt
package com.ssafy.diary.dto

data class Diary(
    var diaryNo: Int,
    var userId: String,
    var diaryEmotion: String,
    var diaryContent: String,
    var diaryDate: String,
    var diaryImg: String
){
    constructor(): this(0, "", "", "", "", "")
}
